src/app/(app)/_layout.tsx

```diff
@@ -102,7 +109,8 @@ export default function TabLayout() {
               renderIcon(icon, color, focused),
             tabBarTestID: testID,
             headerShown,
-            tabBarButton: name === 'product-list' ? () => null : undefined,
+            tabBarButton:
+              name === ('product-list' || 'payment') ? () => null : undefined,
```

Comment:
this line of code is doing the exact same thing that the previous one, did you mean


name === 'product-list' || name === 'payment' ? () => null : undefined, ?